Makefile

```diff
@@ -28,15 +28,16 @@ builddeb-docker: docker-builder-image
 	mkdir -p dist
 	docker run -v $(PWD):/mnt $(DOCKER_BUILDER)
 
-ITEST_TARGETS = itest_xenial itest_bionic itest_stretch itest_buster
+ITEST_TARGETS = itest_bionic itest_focal itest_jammy itest_kinetic itest_bullseye
 
 .PHONY: itest $(ITEST_TARGETS)
 itest: $(ITEST_TARGETS)
 
-itest_xenial: _itest-ubuntu-xenial
 itest_bionic: _itest-ubuntu-bionic
 itest_focal: _itest-ubuntu-focal
 itest_jammy: _itest-ubuntu-jammy
+itest_kinetic: _itest-ubuntu-kinetic
+itest_bullseye: _itest-debian-bullseye
```

Comment:
Personally I would probably leave off kinetic since it's a non-LTS release and Ubuntu only supports those for 9 months.

I think we're probably over-testing a bit in this repo given we build the same simple Debian package from the same build OS regardless of the target OS, but I don't think it really hurts. The context behind that is that we used to have compatibility problems with the `.deb` itself (not the aactivator code) since if we built it on too-new of a release, older versions couldn't install it (due to unsupported compatibility level or compression type or stuff like that). I don't think this has been an issue in recent history but could presumably happen again one day.